src/api/api.tsx

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 import { ERROR_MESSAGES } from '../utilities/constants';
 
 const api = axios.create({
-  baseURL: process.env.REACT_APP_BASE_URL,
```

Comment:
hmm I think we do need to have it as `REACT_APP_BASE_URL` according https://create-react-app.dev/docs/adding-custom-environment-variables/